Library/Formula/glod-cli.rb

```diff
@@ -0,0 +1,30 @@
+require "language/go"
+
+class GlodCli < Formula
+
+  desc "Glod command-line interface tools"
+  homepage "https://github.com/dwarvesf/glod-cli"
+  url "https://github.com/dwarvesf/glod-cli/archive/1.0.1.1.tar.gz"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+
+  go_resource "github.com/kr/fs" do
+    url "https://github.com/kr/fs.git", :revision => "2788f0dbd16903de03cb8186e5c7d97b69ad387b"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GO15VENDOREXPERIMENT"] = "0"
+    mkdir_p buildpath/"src/github.com/dwarvesf/"
+    ln_s buildpath, buildpath/"src/github.com/dwarvesf/glod-cli"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "godep", "go", "build", "-o", "glod-cli", "."
+    bin.install "glod-cli"
+  end
+
+  test do
+    system bin/"glod-cli", "help"
```

Comment:
Hi @DomT4 , I'm a little bit busy with the deadline. This weekend will be good time for new commit